astropy/coordinates/builtin_frames/__init__.py

```diff
@@ -78,37 +78,37 @@
 # get included.  Note that the order here determines the order in the
 # documentation of the built-in frames (see make_transform_graphs_docs).
```

Comment:
The comment here is true, but I don't see why the order should matter. @mhvk, why was it important?

Reply:
good point. Should I exclude the whole subpacakge, or just this file ?

Reply:
That sounds like an oversight and I think it should be reported to Ruff.

Reply:
For now, my tendency would be to just mark this `__all__` with the appropriate `noqa` - that then is also a reminder to fix it at some point.

Reply:
adding a noqa here was surprisingly difficult, but it's done !